packages/fiori/src/UserMenuAccount.ts

```diff
@@ -0,0 +1,81 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import { customElement } from "@ui5/webcomponents-base/decorators.js";
+import property from "@ui5/webcomponents-base/dist/decorators/property.js";
+
+@customElement({
+	tag: "ui5-user-menu-account",
+})
+/**
+ * @class
+ * ### Overview
+ *
+ * The `ui5-user-menu-account` represents an account in the `ui5-user-menu`.
+ *
+ * ### ES6 Module Import
+ * `import "@ui5/webcomponents-fiori/dist/UserMenuAccount.js";`
+ *
+ * @constructor
+ * @extends UI5Element
+ * @experimental
+ * @public
+ * @since 2.5.0-rc.1
+ */
+class UserMenuAccount extends UI5Element {
+	/**
+	 * Defines the avatar of the user.
+	 *
+	 * @default ""
+	 * @public
+	 */
+	@property({ type: String })
+	avatar?: string;
+
+	/**
+	 * Defines the avatar initials of the user.
+	 *
+	 * @default null
+	 * @public
+	 */
+	@property({ type: String })
+	initials = null;
+
+	/**
+	 * Defines the title text of the user.
+	 *
+	 * @default ""
+	 * @public
+	 */
+	@property({ type: String })
+	text = "";
```

Comment:
If the default value is meant to be false in order to not render something and you don't explicitly need it to be string it is preferred to be switch to optional property. Some for next properties of this kind.